Resources/Prototypes/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml

```
@@ -98,7 +98,7 @@
   name: Mateba
   parent: [BaseWeaponRevolver, BaseMajorContraband]
   id: WeaponRevolverMateba
-  description: The iconic sidearm of the dreaded death squads. Uses .45 magnum ammo.
+  description: The iconic sidearm of Nanotrasens Department of Asset Protection. Uses .45 magnum ammo # DeltaV
   components:
```

Possessive is with `'s`

```suggestion
  description: The iconic sidearm of Nanotrasen's Department of Asset Protection. Uses .45 magnum ammo # DeltaV
  components:
```